core/server/src/test/java/alluxio/master/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -77,6 +92,10 @@ public void setUp() {
     when(mMaster.getBlockMaster()).thenReturn(mBlockMaster);
     when(mContext.getAttribute(MasterUIWebServer.ALLUXIO_MASTER_SERVLET_RESOURCE_KEY)).thenReturn(
         mMaster);
+
+    Configuration.set(PropertyKey.UNDERFS_ADDRESS, "test://test");
```

Comment:
Since we're modifying global configuration, we need an `@After` method to reset it before future tests run:

```java
@After
public void after() {
  ConfigurationTestUtils.resetConfiguration();
}
```